app/assets/stylesheets/views/_landing_page.scss

```diff
@@ -0,0 +1,19 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.landing-page-header {
+  // I've used the hex value for the background colour as it doesn't exist in the colour palette
+  background-color: #231F20;
+  width: 100%;
+}
+
+.landing-page-header__org {
+  padding-top: govuk-spacing(2);
+  padding-bottom: govuk-spacing(6);
+
+  // I've added this styling here rather than in the gem because it's application specific
+  // and it would introduce further work to make this variation work in the gem without
+  // affecting the other variants.
+  .gem-c-organisation-logo__link:link:not(:hover) {
+    text-decoration: none;
```

Comment:
Why remove the underline on hover? Did I miss a discussion about the design?

Reply:
This doesn't remove the underline on hover. According to the designs, the link should not have an underline for its default state but I imagine we still want an underline to appear on hover. Basically the above is a one liner for doing something like:

```
.gem-c-organisation-logo__link:link {
    text-decoration: none;

    &:hover {
      // reinstate hover underline styling
    }
  }
```

If this is more readable and preferable, happy to make the change

Reply:
Oh I see. Actually I was more concerned about styling a component outside of a component, which breaks component isolation. For the purposes of getting this over the line can you just put a comment that we need to add a variant of the organisation logo to do this, rather than styling it here? Then we'll sort that separately.